src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/LocalizableStrings.resx

```diff
@@ -125,6 +125,10 @@
     <comment>{0} is a list of single-word identifier strings separated by comma. Such as "myManualInstruction, someUserPickedText, instruction3".
 {1} is a single-word identifier string. Such as "myPostAction".</comment>
   </data>
+  <data name="Authoring_InvalidMultichoiceSymbol" xml:space="preserve">
+    <value>Choice parameter {0} is invalid. It allows multiple values ('AllowMultipleValues=true'), while some of the configured choices contain separator characters. Invalid choices: {1}</value>
+    <comment>{0} is the offending symbol name, {1} is a csv list of offending choice values</comment>
+  </data>
```

Comment:
I like the wording here! It clearly describes the choice parameter that failed validation and why. One suggestion - I think we should include the valid separator character(s) in the message.

```suggestion
  <data name="Authoring_InvalidMultichoiceSymbol" xml:space="preserve">
    <value>Choice parameter {0} is invalid. It allows multiple values ('AllowMultipleValues=true'), while some of the configured choices contain the separator character(s) {1}. Invalid choices: {2}</value>
    <comment>{0} is the offending symbol name, {1} is a set of separator characters, and {2} is a csv list of offending choice values</comment>
  </data>
```

Reply:
Good improvement @baronfel! Incorporating.